pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools","cython", "numpy","scipy"]
\ No newline at end of file
+requires = ["setuptools","cython", "numpy==1.19.2","scipy"]
```

Comment:
should we do >= instead of exact version?

Reply:
Right now the latest version of numpy dies on windows. Probably eventually we want >=, but now we don't have much of a choice